### DISCUSSION TITLE
Add a `FirstJoinMOTD` under the `messages:` tab of the config

### DISCUSSION THREAD

**Discussion by MrMcyeet** [Zrips/CMI#4370] [Feature Requests]
**Description of feature/improvement request:**
When a player joins for the first time, it would display a different motd than normal players get when they re-join the server.

---
### Detailed explanation of what it should do, when and in what situation: ###
When a player joins for the first time, it would display this special motd, vs the normal motd.

### Basic explanation as to why this feature is needed and/or useful to have: ###
Useful for telling players a little bit about the server when they first join, and not spamming every player with the same message every time they login.

**Comment by CloudeLecaw** [comment]
The Motd is shown on the serverchoosing screen
So its not possible for the server to notice someone is there
![Snag_e393923](https://user-images.githubusercontent.com/12080017/95932444-2c245480-0dcc-11eb-84c9-6fac3d52ccf1.png)
As long as you are in this interface where you choose the server, the server has no option to get information about you and mess it with its databse if your account exists or not.

So this is not possible

All Interactions that would be possible here are !Serer notices someone pinged the server and log that or prevent connection if same IP pings to often. But still in this case the server does not know "who" is pinging and if he was there or not, 
May he could log the IP of the ping. but when your IP changes then you would be someone new for the server each time and the server does not know u loged in allready or not.
What the server could do: If you loggin in the same day with same ip the server can see "this ip was here allready this day"
